gulpfile.js

```diff
@@ -46,34 +44,35 @@ gulp.task('pl-clean:public', function() {
 });
 
 // Copy data files from source into public folder:
-gulp.task('pl-copy:data', function () {
+gulp.task('pl-copy:data', function() {
   return gulp.src('source/data/*.json')
     .pipe(gulp.dest(paths().public.root + '/data'));
 });
 
 // Copy jQuery distribution from installed package into public JS folder:
-gulp.task('pl-copy:jq', function () {
+gulp.task('pl-copy:jq', function() {
   return gulp.src('node_modules/jquery/dist/jquery.min.js')
   .pipe(gulp.dest(paths().public.js))
 });
 
 // Copy image files from source into public images folder:
-gulp.task('pl-copy:img', function () {
+gulp.task('pl-copy:img', function() {
   return gulp.src(
     ['**/*.gif', '**/*.png', '**/*.jpg', '**/*.jpeg', '**/*.svg'],
     { cwd: paths().source.images }
   ).pipe(gulp.dest(paths().public.images));
 });
 
 // Copy favicon file from source into public folder:
-gulp.task('pl-copy:favicon', function () {
+gulp.task('pl-copy:favicon', function() {
   return gulp.src('favicon.ico', { cwd: paths().source.root })
     .pipe(gulp.dest(paths().public.root));
 });
 
 // Create flat designsystem CSS file and put into public CSS folder:
-gulp.task('pl-copy:css', function () {
-  return gulp.src(paths().source.css + 'style.scss')
+gulp.task('pl-copy:css', function(done) {
+  buildConfig.production.forEach(function(element) {
```

Comment:
This one iterates over stylesheets in the production object, not the dev object in the config.

Reply:
Fixed it